assets/css/sensei-banner.scss

```diff
@@ -0,0 +1,106 @@
+
+@media screen and (max-width: 780px) {
+	.woocommerce-banner {
+		display: flex;
+		flex-direction: column;
+	}
+}
+
+@media screen and (min-width: 1100px) {
+	.woocommerce-banner {
+		display: flex;
+		flex-direction: row;
+	}
+}
+
+#woothemes-sensei .woocommerce-banner {
+	&__body {
+		color: #FFFFFF;
+		font-size: 16px;
+		font-style: normal;
+		font-weight: 400;
+		line-height: 26px;
+		letter-spacing: 0em;
+		text-align: left;
+		margin-bottom: 20px;
+
+	}
+
+	&__header {
+		color: #6fcfb2;
+		font-family: 'Reforma 1918-Blanca';
```

Comment:
I'm not sure if the font is loaded. Also, maybe you should move the font file from `fonts/sensei-banner/Reforma1918-Blanca.otf` to `fonts/Reforma1918-Blanca.otf` because it will be used on other places as well.

Reply:
Yes, totally forgot 
https://github.com/Automattic/sensei/pull/4801/files#diff-26a9c672515c07aeff464f58097c6a2af395a752d7c4e3353e0b81a1925d9acdR81